src/lib/sql/sql_translator.cpp

```diff
@@ -191,20 +194,28 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_statement(const hsql:
 
 std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_select_statement(const hsql::SelectStatement& select) {
   // SQL Orders of Operations
-  // 1. FROM clause (incl. JOINs and sub-SELECTs that are part of this)
-  // 2. SELECT list (to retrieve aliases)
-  // 3. WHERE clause
-  // 4. GROUP BY clause
-  // 5. HAVING clause
-  // 6. SELECT clause (incl. DISTINCT)
-  // 7. UNION clause
-  // 8. ORDER BY clause
-  // 9. LIMIT clause
+  // 1. WITH clause
+  // 2. FROM clause (incl. JOINs and sub-SELECTs that are part of this)
+  // 3. SELECT list (to retrieve aliases)
+  // 4. WHERE clause
+  // 5. GROUP BY clause
+  // 6. HAVING clause
+  // 7. SELECT clause (incl. DISTINCT)
+  // 8. UNION clause
+  // 9. ORDER BY clause
+  // 10. LIMIT clause
 
   AssertInput(select.selectList, "SELECT list needs to exist");
   AssertInput(!select.selectList->empty(), "SELECT list needs to have entries");
   AssertInput(!select.unionSelect, "Set operations (UNION/INTERSECT/...) are not supported yet");
 
+  // Translate WITH clause
```

Comment:
clause**s**, right?

Reply:
I thought of the WITH clause as a whole with multiple statements or descriptions: 
```SQL
WITH
a AS (..), 
b AS (..), 
c AS (..)
```
But this is my interpretation.

Reply:
Ah, I was mistaken about the syntax. If there is ever only one WITH, leave it at "clause"